waku/waku_lightpush/protocol.nim

```diff
@@ -23,7 +24,7 @@ logScope:
 type WakuLightPush* = ref object of LPProtocol
   rng*: ref rand.HmacDrbgContext
   peerManager*: PeerManager
-  pushHandler*: PushMessageHandler
+  wakuRelay*: WakuRelay
```

Comment:
This means that to test `WakuLightPush` you need `WakuRelay`. Please, no.

IMO, only acceptable options are;
- some kind of interface (more boilerplate)
- a channel (not yet functional in Chronos)
- a callback (current solution)